.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+## Jira Issue Link
+
```

Comment:
Is it ok to link to our internal JIRA?

Reply:
Good point - maybe just ask for the ticket number? I think we need some way to know what ticket is involved in case the branch isn't named with the ticket number.

Reply:
Changed to ## Jira Issue ID